app/modules/listings/components/listing_element.svelte

```diff
@@ -23,31 +33,90 @@
   } else if (image?.url) {
     imageUrl = image.url
   }
+
+  function toggleShowMode () {
+    isShowMode = !isShowMode
+    if (isShowMode) {
+      const options = { metadata: getElementMetadata(listing, element), preventScrollTop: true }
+      app.navigate(getElementPathname(listingId, elementId), options)
+    } else {
+      const options = { metadata: getListingMetadata(listing), preventScrollTop: true }
+      app.navigate(getListingPathname(listingId), options)
+    }
+  }
+
+  $: comment = element.comment
 </script>
-<div class="listing-element-section">
+{#if isShowMode}
+  <Modal on:closeModal={toggleShowMode}
+  >
```

Comment:
The modal is unnecessarily narrow on large screens, especially noticeable when there is a lot of info
```suggestion
  <Modal on:closeModal={toggleShowMode} size="large">
```

Reply:
done in d3d025a7c

Reply:
actually, i think it needs a more refined size customization
Small screens looks too narrow:
![2024-10-17_15-01](https://github.com/user-attachments/assets/34dfad06-39c4-4ebf-9906-79a8d3fc01ae)

Reply:
I wish we had a linter rule for this, but using `:global` at the component root should be forbidden, as it's then unscopped and can thus leak to affect other parts of the app. My alternative proposal: revert fd9de6e and pick ac42be86c from the branch [`listing-element-comment-max`](https://github.com/inventaire/inventaire-client/tree/listing-element-comment-max)

Reply:
nicely spotted, this was a mistake. I pushed a fixup commit 99a01ff03 reusing the rules i defined earlier since i would like to keep the `40em` since its the size of the lisitng elements in the background. And also prefering widely existing rules (@media queries) than your proposal.

Reply:
I'm not really convinced, but let's give it a shot like this and see if it stays itchy